lighthouse-cli/test/smokehouse/test-definitions/a11y/expectations.js

```diff
@@ -30,6 +30,15 @@ const expectations = [
                   'nodeLabel': 'div',
                 },
               },
+              {
+                node: {
+                  'type': 'node',
+                  'selector': 'body > section > div#duplicate-id-aria > div',
+                  'snippet': '<div aria-labelledby="duplicate-id-aria">',
+                  'explanation': 'Fix all of the following:\n  ARIA attribute cannot be used, add a role attribute or use a different element: aria-labelledby',
+                  'nodeLabel': 'div',
+                },
+              },
```

Comment:
use `<input type="text" aria-labelledby="duplicate-id-aria"/>` like https://github.com/GoogleChrome/lighthouse/pull/12536/files#diff-ac2a6c87b89f6ef634950dc38d9b64028ebd23611a031bedb1340254c13b8d11R136

in a11y_tester to get rid of this